### ISSUE TITLE
 What data set did you use to train the InceptionV3 model?

### ISSUE THREAD

**Issue by xinchunl** [openimages/dataset#23]
I notice that human-annotations labels.csv contains ~1.7M #samples, but in train/images.csv there's ~9M #samples. I assume they cannot fully join with labels file. 
So I wonder you used the machine-annotations as training data set? or some else labels could join training set? 

furthermore, for the unbalanced #samples between categories, how do you handle this? down or up sampling? or just leave them as many as they have? 

Thanks, 
XL

**Comment by gkrasin** [comment]
@xinchunl the classifier was trained on machine image-level annotations available for download from the [main page](https://github.com/openimages/dataset). The machine annotation is an output of another classifier internal to Google.

It's expected that there will be more annotations (including human rated) in both train and validation sets eventually.

The unbalanced story is trivial for the classifier released: we did nothing. That's why many of the rare labels don't fire much and the output needs to be calibrated to get at least some reasonable values on them. We believe that there are multiple possible approaches are viable to handle this, you're welcome to try and report the results. :)

**Comment by xinchunl** [comment]
Thanks Ivan for your explanation.
